components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/bindings/handlers/TokenBindingExpiryEventHandler.java

```diff
@@ -100,6 +113,124 @@ public void handleEvent(Event event) throws IdentityEventException {
         }
     }
 
+    /**
+     * This method will get the application information from session context and revoke access tokens of the
+     * applications bound to that session. This method can be used when token binding information id not found in the
+     * request.
+     *
+     * @param event Event.
+     * @throws IdentityOAuth2Exception
+     * @throws InvalidOAuthClientException
+     */
+    private void revokeAccessTokensForSSOSessionBinding(Event event)
+            throws IdentityOAuth2Exception, InvalidOAuthClientException {
+
+        String sessionContextIdentifier = getSessionIdentifier(event);
+        Map<String, Object> eventProperties = event.getEventProperties();
+        if (StringUtils.isNotBlank(sessionContextIdentifier)) {
+            SessionContext sessionContext = (SessionContext) eventProperties.get(IdentityEventConstants
+                    .EventProperty.SESSION_CONTEXT);
+            if (sessionContext != null) {
+                Map<String, SequenceConfig> authenticatedSequences = sessionContext.getAuthenticatedSequences();
+                if (MapUtils.isEmpty(authenticatedSequences)) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("Could not find the authenticated sequence config map in the session context");
+                    }
+                    return;
+                }
+
+                AuthenticatedUser user = (AuthenticatedUser) sessionContext
+                        .getProperty(FrameworkConstants.AUTHENTICATED_USER);
+                if (user == null) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("Authenticated user is nnot founnd in the session context");
```

Comment:
"Authenticated user is **nnot founnd** in the session context" -> typo in the message.
Also, add some context information to the log for it to be meaningful.